tests/AlyxPanel_test.m

```diff
@@ -412,19 +412,19 @@ function test_giveWater(testCase)
         testCase.SubjectUI.Selected, datestr(now, 'yyyy-mm-dd'),datestr(now, 'yyyy-mm-dd'));
       [vals, record] = get_test_data;
       
-      testCase.verifyEqual(record.expected_water, vals(2), 'RelTol', 0.1, 'Expected water mismatch')
```

Comment:
you could just have a `tol` variable that you define at the top of this function so you don't have to change each of these individually? i.e. avoid magic numbers